src/ui/Notification.tsx

```diff
@@ -50,7 +50,6 @@ const Notification = ({ icon, header, body }: Props) => {
         paddingBottom={4}
         paddingRight={5}
         paddingLeft={4}
-        as={BootstrapToast.Body}
```

Comment:
Does it still render as a bootstrap toast now?

Reply:
Yes the body is actually just a forwarded `div` I think it's there more for semantic purposes

Reply:
Of course I say that but now the tests fail on it -.- Will see if I can use the Body without it complaining about refs